File: src/main/java/net/dashflight/data/jwt/create/DashflightJwtCreatorModule.java
package net.dashflight.data.jwt.create;

import com.google.inject.AbstractModule;
import java.util.Random;
import net.dashflight.data.jwt.create.request.DashflightCreateJwtRequestProviderModule;
import net.dashflight.data.random.LavaRandom;

public class DashflightJwtCreatorModule extends AbstractModule {

    @Override
    protected void configure() {
        install(new DashflightCreateJwtRequestProviderModule());

        bind(JwtCreator.class);
        bind(Random.class).to(LavaRandom.class);
    }

}

File: src/main/java/net/dashflight/data/jwt/create/DashflightJwtCreatorModule.kt
package net.dashflight.data.jwt.create

import com.google.inject.AbstractModule
import net.dashflight.data.jwt.create.request.DashflightCreateJwtRequestProviderModule
import net.dashflight.data.random.LavaRandom
import java.security.SecureRandom
import java.util.*

class DashflightJwtCreatorModule : AbstractModule() {

    override fun configure() {
        install(DashflightCreateJwtRequestProviderModule())

        bind(JwtCreator::class.java)
        bind(Random::class.java).to(SecureRandom::class.java)
    }
}